core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -121,6 +139,37 @@
    */
   void fetchColumn(Text colFam, Text colQual);
 
+  /**
+   * Adds a column to the list of columns that will be fetched by this scanner. The column is
+   * identified by family and qualifier. By default when no columns have been added the scanner
+   * fetches all columns.
+   *
+   * <p>
+   * <b>WARNING</b>. Using this method with custom iterators may have unexpected results. Iterators
```

Comment:
Could say see the warning on {@link #fetchColumn(Text, Text)}

Reply:
Fixed. See 787590acc34707